deltachat-ios/Controller/QrCodeReaderController.swift

```diff
@@ -186,6 +202,15 @@ class QrCodeReaderController: UIViewController {
     func stopSession() {
         captureSession.stopRunning()
     }
+
+    @objc private func moreButtonPressed() {
+        let alert = UIAlertController(title: title, message: nil, preferredStyle: .safeActionSheet)
```

Comment:
ftr: as discussed one-to-one: for consistency with the first device, no need to have "troubleshooting" that much in the face (things shall work - otherwise, ppl will find the option:), "Troubleshooting" is clearer than just "?/Help"